consensus/src/twins_test.rs

```diff
@@ -162,24 +183,42 @@ impl SMRNode {
             config.base.waypoint = WaypointConfig::FromConfig(waypoint);
             config.consensus.proposer_type = proposer_type;
             // Use in memory storage for testing
-            // node_config.consensus.safety_rules = SafetyRulesConfig::default();
+            //config.consensus.safety_rules = SafetyRulesConfig::default();
```

Comment:
Remove?

Reply:
Removed.

Reply:
I think it's still there @sheharbano  =)

Reply:
Oops, sorry! (removed it locally but forgot to commit)